kedro-org/kedro-viz
# Fix: lambda python functions not rendering correctly on flowchart

## Description

Fixes #817.

## Development notes

I'm no longer using `_strip_tags` in the backend, so the `<>` come through to the frontend. I've create a `replaceMatches()` utility function and added that to the `node-list-row.js` file to ensure both `<lambda>` and `<partial>` will render via `dangerouslySetInnerHTML`.

## QA notes

I've added a lambda node so this can be tested. Open the Gitpod URL and expand the `Ingestion` modular pipeline. You'll see the lambda function there.

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/851"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
